### PR TITLE
Add company and socials to backoffice

### PR DESCRIPTION
**Pull request by beasrodrigues24**
Why:
* For it to be possible to add companies, select the company using the wheel and link socials to the companies

How:
* By generating the admin resources

### CODE REVIEW

File: app/admin/companies.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Company do
+  # See permitted parameters documentation:
```
Comment (zamith) [inline app/admin/companies.rb:4]: Do you need these comments?

File: app/controllers/companies_controller.rb [hunk 2]
```diff
@@ -7,8 +7,8 @@ def index
   end
 
   def serialize_company(company)
-    serialized_socials = Social.where(company_id: company.id).map do |social|
-      social.slice(:name, :url)
+    serialized_socials = Social.where(company_id: company.id).all.each_with_object({}) do |social, acc|
```
Comment (zamith) [inline app/controllers/companies_controller.rb:10]:
```suggestion
    serialized_socials = company.socials.each_with_object({}) do |social, acc|
```

What about this?

Comment (beasrodrigues24) [reply app/controllers/companies_controller.rb:10]: Will fix in the other PR and rebase :smile:

File: app/admin/companies.rb (hunk 1 above)
Comment (beasrodrigues24) [reply app/admin/companies.rb:4]: What comment to you mean? The documentation or the frozen_string_literal? If it's the second rubocop forces me to have that. If it's the documentation I can remove it

Comment (zamith) [reply app/admin/companies.rb:4]: The documentation. I understand it's auto generated, but if you don't need it, remove it.

**Review by zamith — APPROVED** [review]
Would remove the comments, but it's not a blocker

**Pull request merged**